api/msg.go

```diff
@@ -249,6 +249,18 @@ func (s *ledgerStatusResponse) marshalJSON(arena *fastjson.Arena) ([]byte, error
 	return o.MarshalTo(nil), nil
 }
 
+type nonceResponse struct {
```

Comment:
This struct should export its fields to make it consistent with other structs. It should also have `json` tags for the same reason.